iceoryx_posh/include/iceoryx_posh/popo/untyped_subscriber.hpp

```diff
@@ -48,13 +48,43 @@ class UntypedSubscriberImpl
     using BaseSubscriber::getServiceDescription;
     using BaseSubscriber::getSubscriptionState;
     using BaseSubscriber::getUid;
-    using BaseSubscriber::hasMissedSamples;
-    using BaseSubscriber::hasSamples;
+    using BaseSubscriber::hasMissedSamples; // iox-#408 remove
+    using BaseSubscriber::hasSamples;       // iox-#408 remove
     using BaseSubscriber::invalidateTrigger;
-    using BaseSubscriber::releaseQueuedSamples;
+    using BaseSubscriber::releaseChunk;
+    using BaseSubscriber::releaseQueuedSamples; // iox-#408 remove
     using BaseSubscriber::subscribe;
-    using BaseSubscriber::take;
+    using BaseSubscriber::take; // iox-#408 replace
     using BaseSubscriber::unsubscribe;
+
+    // iox-#408
+    // the 1_0 suffix is only used temporarily to not cause regressions in all examples and tests and keep the changes
+    // as small as possible, it will replace the function without suffix in a follow-up pull request (which changes
+    // all examples)
+
+    cxx::expected<const void*, ChunkReceiveResult> take_1_0() noexcept;
+
+    // iox-#408
+    // the untyped API is supposed to deal with chunks, hence the renaming iox #408 remove comment
+    // calling it chunks looks inappropriate in the function names (use data instead of chunks?)...
+
+    ///
+    /// @brief hasChunks Check if chunks are available.
+    /// @return True if a new chunk is available.
+    ///
+    bool hasChunks() const noexcept;
```

Comment:
I was also thinking about better names for the functions in the untyped case, maybe hasData? hasChunks does not look right, but neither does hasSamples in the untyped case. I am open for suggestions but also do not consider it a major problem.

Reply:
since all the other methods also call it `Chunks` I would keep it for consistency reasons or rename everything to `*Data`

Reply:
I understood that we wanted the users of the untyped API to actually be thinking in terms of memory chunks when reasoning about the logic.
See: https://github.com/eclipse-iceoryx/iceoryx/issues/408#issuecomment-757910091

Reply:
I got the same impression @ithier , but I am open for proposals. When there are none, I will leave it at chunks.